static/css/components/dropper.less

```diff
@@ -195,7 +195,7 @@ div.Tools {
   // And Orphaned Editions (no work)
   .widget-add.old-style-lists {
     .dropclick {
-      padding: 10px;
+      padding: 5px;
```

Comment:
Is this needed?  This seems to be unrelated to this PR.

Reply:
It's related to the other issue this PR closes https://github.com/internetarchive/openlibrary/issues/4987.